demo/app/components/demo-map/demo-map.component.ts

```diff
@@ -1,46 +1,85 @@
-import { Component, ViewChild } from '@angular/core';
+import { AfterViewInit, Component, ViewChild } from '@angular/core';
+import { ZoomToRectangleService } from '../../../../src/angular-cesium-widgets/services/zoom-to-rectangle.service';
+import { MapLayerProviderOptions } from '../../../../src/angular-cesium/models/map-layer-provider-options.enum';
+import { SceneMode } from '../../../../src/angular-cesium/models/scene-mode.enum';
 import { ViewerConfiguration } from '../../../../src/angular-cesium/services/viewer-configuration/viewer-configuration.service';
 import { AppSettingsService } from '../../services/app-settings-service/app-settings-service';
-import { MapLayerProviderOptions } from '../../../../src/angular-cesium/models/map-layer-provider-options.enum';
 import { TracksLayerComponent } from '../tracks-layer/tracks-layer.component';
-import { SceneMode } from '../../../../src/angular-cesium/models/scene-mode.enum';
 
 @Component({
-	selector : 'demo-map',
-	templateUrl : './demo-map.component.html',
-	providers : [ViewerConfiguration],
-	styleUrls : ['./demo-map.component.css']
+  selector: 'demo-map',
+  templateUrl: './demo-map.component.html',
+  providers: [ViewerConfiguration],
+  styleUrls: ['./demo-map.component.css'],
 })
-export class DemoMapComponent {
-	@ViewChild('layer') tracksLayer: TracksLayerComponent;
-	arcGisMapServerProvider = MapLayerProviderOptions.ArcGisMapServer;
-	sceneMode = SceneMode.SCENE3D;
-	
-	constructor(private viewerConf: ViewerConfiguration,
-							private appSettingsService: AppSettingsService) {
-		viewerConf.viewerOptions = {
-			selectionIndicator : false,
-			timeline : false,
-			infoBox : false,
-			fullscreenButton : false,
-			baseLayerPicker : false,
-			animation : false,
-			shouldAnimate: false,
-			homeButton : false,
-			geocoder : false,
-			navigationHelpButton : false,
-			navigationInstructionsInitiallyVisible : false,
-		};
-		
-		viewerConf.viewerModifier = (viewer: any) => {
-			viewer.screenSpaceEventHandler.removeInputAction(Cesium.ScreenSpaceEventType.LEFT_DOUBLE_CLICK);
-			viewer.bottomContainer.remove();
-		};
-		
-		this.appSettingsService.showTracksLayer = true;
-	}
-	
-	removeAll() {
-		this.tracksLayer.removeAll();
-	}
+export class DemoMapComponent implements AfterViewInit {
+  @ViewChild('layer') tracksLayer: TracksLayerComponent;
+  arcGisMapServerProvider = MapLayerProviderOptions.ArcGisMapServer;
+  sceneMode = SceneMode.SCENE3D;
+  showLayer = true;
+  showMap = true;
+  maps = [
```

Comment:
why there is an array here? its confusing 
If we want an example for multi ac-map lets create a new component for it... after all multi map is used that often... and the demo has enough code without it...